examples/expression-equality/expression_equality.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/io;
+
+public function main() {
+    map<string> student = {"name": "John", "age": "25"};
+    map<anydata> student2 = {"name": "John", "age": "25"};
+
+    // The output will be `true` because the values are considered equal based on the
+    // equality of members.
+    io:println(student == student2);
+
+    // The output will be false because the values are not equal due to differences in their content.
```

Comment:
Is this reason correct? Here the content is equal in the given 2 values right?

Reply:
Updated